testing/testing.h

```diff
@@ -23,6 +24,8 @@ fml::UniqueFD OpenFixturesDirectory();
 
 fml::UniqueFD OpenFixture(std::string fixture_name);
 
+std::unique_ptr<fml::Mapping> OpenFixtureAsMapping(std::string fixture_name);
```

Comment:
Consider documenting this (and the surrounding prototypes while you're in here).

Reply:
Done.